cdc-accelerator/feature/acls.js

```diff
@@ -0,0 +1,70 @@
+import PartnerFeature from './partnerFeature.js'
+import path from 'path'
+import fs from 'fs'
+import { clearDirectoryContents } from '../utils/utils.js'
+import { SRC_DIRECTORY, BUILD_DIRECTORY } from './constants.js'
+import client from '../../cdc-accelerator/sap-cdc-toolkit/gigya/client.js'
+export default class ACL extends PartnerFeature {
+    static ACL_FILE_NAME = 'Acls.json'
+    constructor(credentials) {
+        super(credentials)
+    }
+
+    getName() {
+        return this.constructor.name
+    }
+
+    async init(permissionGroupsFile, partnerId, partnerDirectory, dataCenter) {
```

Comment:
ACL should not know anything about PermissionGroups, it should not receive as argument the permissionGroups file.